contracts/src/gateway/router/TopDownFinalityFacet.sol

```diff
@@ -35,6 +35,15 @@ contract TopDownFinalityFacet is GatewayActorModifiers {
         s.validatorsTracker.batchStoreChange(changeRequests);
     }
 
+    /// @notice Get the configuration numbers in the validatorTracker struct that shadows the configuration changes
+    /// from the parent in the child gateway
```

Comment:
```suggestion
    /// @notice Get the next and start configuration numbers in the tracker of changes
    /// from the parent in the child gateway
```

Nit: clarity (explain 2 return values), abstraction (avoid leaking internal structure). It's still not super clear but probably makes sense in context.